packages/common/src/surveys/HubSurvey.ts

```diff
@@ -37,6 +38,33 @@ export class HubSurvey
     return new HubSurvey(pojo, context);
   }
 
+  /**
+   * Fetch a Survey from the backing store and return a HubSurvey instance.
+   * @param identifier - Identifier of the survey to load
+   * @param context
+   * @returns
+   */
+  static async fetch(
```

Comment:
Thanks for this update! I don't remember why we didn't initially have a fetch function for surveys...

Reply:
Opendata-ui fetches entities using the [fetchHubEntity](https://github.com/Esri/hub.js/blob/f/7511-event-entity/packages/common/src/core/fetchHubEntity.ts) facade method which bypasses the entity classes altogether and delegates directly to the entity's fetch method in their respective `fetch` modules, so it technically didn't hurt anything for this _not_ to be implemented. I simply added it for consistency.